frontend/src/components/Contact.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import "./Contact.scss";
+
+class Contact extends React.Component {
+  constructor(props) {
+    super(props);
+
+    // Value allocation displayed basic ********
```

Comment:
These comments don't seem mandatory to me, they don't *explain* the code below, they merely describe it